### PR TITLE
Inital Contributing guide

### PR DESCRIPTION
**Pull request by rhyshort**
Initial contributing guide, inspired by sync-android's contributing.md

reviewer @mikerhodes 
reviewer @brynh

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+Contributing
+=======
+
+Cloudant-client is written in Java and uses maven as it's build tool.
```
Comment (brynh) [inline CONTRIBUTING.md]: Typo:  "it's" should be "its".

Comment (rhyshort) [inline CONTRIBUTING.md]: done in 5b0c0e4

**Comment by brynh** [comment]
:+1:

**Comment by mikerhodes** [comment]
Remove the coding guidelines for now until we have a chance to fix up the whole repo.

**Comment by mikerhodes** [comment]
Also, perhaps improve the docs so you don't have to have the .sample properties files.

**Comment by rhyshort** [comment]
I think we leave it for now, and make the docs better when we make it possible to configure test via system properties like sync-android

**Comment by mikerhodes** [comment]
Cool :+1:

**Pull request merged**